packages/twenty-docker/k8s/manifests/deployment-server.yaml

```diff
@@ -31,6 +31,10 @@ spec:
           value: 3000
         - name: SERVER_URL
           value: "https://crm.example.com:443"
+        - name: FRONT_BASE_URL
+          value: "https://crm.example.com:443"
+        - name: BACKEND_SERVER_URL
```

Comment:
actually, I am not sure this one is useful, it is not used in the codebase anymore